### PR TITLE
fix(disrupt_nodetool_enospc): re-enable enospc nemesis

### PR DESCRIPTION
**Pull request by roydahan**
Since Scylla issue #11245 is now considered as fixed, we would like to enable this nemesis so we can continue testing it.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by roydahan** [comment]
Backports should be done ONLY after we make sure the fix in master works and the nemesis doesn't produce a coredump.

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -1529,8 +1529,6 @@ def disrupt_nodetool_refresh(self, big_sstable: bool = False):
             assert '(1 rows)' in result.stdout, f'The key {key} is not loaded by `nodetool refresh`'
 
     def disrupt_nodetool_enospc(self, sleep_time=30, all_nodes=False):
-        raise UnsupportedNemesis('disabling ENOSPC nemesis until #11245 will be fixed')
-
         # pylint: disable=unreachable
```
Comment (fgelcer) [inline sdcm/nemesis.py:1532]: the following line also must be removed:
```
# pylint: disable=unreachable
```

**Review by fgelcer — CHANGES_REQUESTED** [review]

**Review by fgelcer — APPROVED** [review]
LGTM

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**